_posts/2022-12-05-trino-summit-2022-goldman-sachs-recap.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Leveraging Trino to Power Data at Goldman Sachs"
+author: "Sumit Halder, Siddhant Chadha, Suman-Newton, Ramesh Bhanan, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/goldman-sachs.png
+---
+
+Continuing with [the Trino Summit 2022 sessions posts]({% post_url
+2022-11-21-trino-summit-2022-recap %}), we're diving into an insightful
+lightning talk from [Goldman Sachs](https://www.goldmansachs.com) about how they
+use Trino to ensure data quality across the board for all users and customers.
+By using Trino to federate their various data sources, querying everything in
+one place provides them with the flexibility they need to validate that all data
+is as it should be where that data lives, assuaging any concerns that may exist
+about data integrity.
+
+<!--more-->
+
+{% youtube g9fLA3tFG-Q %}
+
+## Recap
+
+Validating data quality can be a tricky and complicated process. Data resides
+in many sources, with different rules and different processes for checking
+quality, and Goldman's data ingestion team may not have a detailed understanding
+of all data sets. Despite that, there is a need to autonomously verify and
+validate all data to be confident in its quality and integrity. The solution to
+this challenge? A queryable data quality platform powered by Trino.
+
+The underlying data quality platform's logic handles the validation, and resting
+on top of it is Trino, the scalable, fast solution to ensure that users can
+query what they need. Even when the platform is profiling the data, enforcing
+various quality rules, and validating the data in different ways, Trino is there
```

Comment:
```suggestion
various quality rules, and validating the data in different ways, Trino is there.
```

Reply:
I like leaving this as-is, I think breaking it up into two sentences loses meaning, and we've shortened all the sentences before this, so we can afford to have a longer one. Writing works and flows better when sentence length varies.